src/core/connection.c

```diff
@@ -178,6 +178,12 @@ QuicConnAlloc(
                     ((uint8_t*)&Datagram->Route->LocalAddress.Ipv6.sin6_addr) + 12,
                     4);
             }
+        } else if (MsQuicLib.Settings.LoadBalancingMode == QUIC_LOAD_BALANCING_SERVER_ID_FIXED) {
+            CxPlatRandom(1, Connection->ServerID); // Randomize the first byte.
```

Comment:
Is "fixed" an industry term here? Because it's pretty confusing to call something "fixed" when it has a random component.

Reply:
The first byte is randomized. The rest is fixed.

Reply:
k XD